assets/sass/_general.scss

```diff
@@ -88,6 +88,15 @@
   margin-bottom: govuk-spacing(4);
 }
 
+.black-background .govuk-hub-comment-form .govuk-fieldset__legend,
+.black-background .govuk-hub-comment-form .govuk-label {
+  color: govuk-colour('white');
+}
+
+.black-background #feedback-widget .govuk-link {
```

Comment:
should this be in a component specific style sheet and not _general.scss as this style is targeting a specific element/component.